extensions/sql-database-projects/src/dialogs/updateProjectFromDatabaseDialog.ts

```diff
@@ -142,7 +140,7 @@ export class UpdateProjectFromDatabaseDialog {
 
 			let formModel = this.formBuilder.component();
 			await view.initializeModel(formModel);
-			this.connectionButton?.focus();
+			await this.connectionButton?.focus();
```

Comment:
this isn't necessary unless this isn't the first focusable element. Alan made a change so that the focus is automatically on the first focusable element in dialogs when they're opened

Reply:
The first focusable element is the textbox for the server name; we want it to be the button for selecting a connection.